ldap/servers/slapd/back-ldbm/archive.c

```diff
@@ -270,6 +270,8 @@ ldbm_back_ldbm2archive(Slapi_PBlock *pb)
     int run_from_cmdline = 0;
     Slapi_Task *task;
     struct stat sbuf;
+    Object *no_object = (Object*)-1;
+    Object *last_busy_inst_obj = no_object;
```

Comment:
I feel that something like this would be a bit cleaner:

```
/* Somewhere earlier in the file */
#define NO_OBJECT (Object*)-1

....

/* Your code */
Object *last_busy_inst_obj = NO_OBJECT;
...
```

This way it can be cleanly reused later if needed by other functions. But it's a minor concern, not a strong objection.